fix: `post-website-push-image-k8s-website-hugo` buildx error

**This is a Bug Report**

**Problem:**

#56563 (`manifest unknown` on `make container-serve`) is caused by [`post-website-push-image-k8s-website-hugo`](https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/k8s-staging-sig-docs.yaml) failing on **every run since at least 2026-05-13** — no Hugo preview image has been pushed in two months, so no locally-computed tag exists anywhere.

Every run fails identically:
```
make: /root/.docker/cli-plugins/docker-buildx: No such file or directory
make: *** [Makefile:103: docker-push] Error 127
```
Evidence: [2026-05-13 run](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/post-website-push-image-k8s-website-hugo/2054619643242352640) · [2026-06-21 run](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/post-website-push-image-k8s-website-hugo/2068722522836176896) (latest available, still failing)

**Root cause:** [`cloudbuild.yaml`](https://github.com/kubernetes/website/blob/main/cloudbuild.yaml) hardcodes `DOCKER_BUILDX=/root/.docker/cli-plugins/docker-buildx` (added [2022-12-05](https://github.com/kubernetes/website/commit/04708a1a83)). When the builder image was bumped on [2026-05-23](https://github.com/kubernetes/website/commit/7bcdcd5c4b) (the old pinned tag had itself vanished from the registry), the new image's buildx plugin landed at a different path, so that hardcoded value has resolved to nothing since. [`contributor-site`'s `cloudbuild.yaml`](https://github.com/kubernetes/contributor-site/blob/main/cloudbuild.yaml) has no such override and works fine.

**Proposed Solution:**

Remove the `DOCKER_BUILDX` line from `cloudbuild.yaml`, matching contributor-site, so `docker-push` falls back to plain `docker buildx` (Docker's own plugin discovery) instead of one hardcoded path:

```diff
     env:
       - DOCKER_CLI_EXPERIMENTAL=enabled
       - DOCKER_BUILDKIT=1
-      - DOCKER_BUILDX=/root/.docker/cli-plugins/docker-buildx
       - TAG=$_GIT_TAG
       - BASE_REF=$_PULL_BASE_REF
```

Related: #56563, #49460, #49610, #49624

--- a/cloudbuild.yaml
+++ b/cloudbuild.yaml
@@ -9,12 +9,11 @@ options:
   machineType: 'E2_HIGHCPU_8'
 steps:
     # It's fine to bump the tag to a recent version, as needed
-  - name: "gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20260108-7f313c340e@sha256:4d778a001ae3f4247b2b61d8a870319e71d66c87556969a6399b90972e4d0491"
+  - name: "gcr.io/k8s-staging-test-infra/gcb-docker-gcloud@sha256:ff388e0dc16351e96f8464e2e185b74a7578a5ccb7a112cf3393468e59e6e2d2" # v20260205-38cfa9523f
     entrypoint: 'bash'
     env:
       - DOCKER_CLI_EXPERIMENTAL=enabled
       - DOCKER_BUILDKIT=1
-      - DOCKER_BUILDX=/root/.docker/cli-plugins/docker-buildx
       - TAG=$_GIT_TAG
       - BASE_REF=$_PULL_BASE_REF
     args:

